### ISSUE TITLE
arduino: incomplete data files are included

### ISSUE THREAD

**Issue by the-maldridge** [voidlinux/void-packages#3637]
The current arduino package is broken and does not launch.  The errors produced on startup are as follows:

```
maldridge@earth:~$ arduino
Picked up JAVA_TOOL_OPTIONS: 
java.io.FileNotFoundException: /usr/lib/arduino/hardware/platform.keys.rewrite.txt (No such file or directory)
WARNING: Error loading hardware folder /usr/lib/arduino/hardware/arduino
  No valid hardware definitions found in folder arduino.
java.io.FileNotFoundException: /usr/lib/arduino/hardware/platform.keys.rewrite.txt (No such file or directory)
java.lang.IllegalStateException: Menu has no enabled items
	at processing.app.Base.selectFirstEnabledMenuItem(Base.java:1571)
	at processing.app.Base.rebuildBoardsMenu(Base.java:1421)
	at processing.app.Base.<init>(Base.java:296)
	at processing.app.Base.guardedMain(Base.java:221)
	at processing.app.Base.main(Base.java:135)
```

Upon furthur inspection the specified files indeed do not exist.

**Comment by ebfe** [comment]
The checksum in the template also doesn't match. Maybe packaged from an incomplete tar ball?

**Comment by ebfe** [comment]
Fixed by #3960
